azure-stack/hci/manage/arc-extension-management.md

```diff
@@ -43,12 +43,57 @@ You can install extensions from the **Capabilities** tab for your Azure Stack HC
 
 When you install an extension in the Azure portal, it's a cluster-aware operation. The extension is installed on all servers of the cluster. If you add more servers to your cluster, all the extensions installed on your cluster are automatically added to the new servers.
 
+
+
 ## Check the extension status
+# [Azure portal](#tab/azureportal)
+After you install an extension, check the status of the extension on each of your nodes. To check the extension status, go to the **Extensions menu page** and view the **status column** of the grid.
 
 You can check the status of an extension on each server from the **Extensions** page by viewing the **status** column of the grid.
 
+
 :::image type="content" source="media/arc-extension-management/arc-extension-status-view.png" alt-text="Screenshot of the different extension statuses in the Azure portal." lightbox="media/arc-extension-management/arc-extension-status-view.png":::
 
+# [Azure CLI](#tab/azurecli)
```

Comment:
The heading following an H2 (##) should be a H3 (###) or H4 (####) as mentioned previously.